ds13/code/modules/mob/living/psychosis_new.dm

```diff
@@ -0,0 +1,106 @@
+#define PSYCHOSIS_EFFECT_LAUGHTER 1
+#define PSYCHOSIS_EFFECT_HITSELF 2
+#define PSYCHOSIS_EFFECT_TRIPPY 3
+#define PSYCHOSIS_EFFECT_WEIRDSCREEN 4
+
+/datum/extension/psychosis
```

Comment:
i don't see anywhere in this code to indicate how this extension is applied, where's the interface for things to be scary